examples/inflow_lognormal.jl

```diff
@@ -56,7 +56,7 @@ gas = Model([1, 2, 11, 12], [1, 2, 11, 12], LogNormal, 0.0; time_varying_params
 initial_params = dynamic_initial_params(y_train, gas)
 
 # Estimate the model via MLE
-fit!(gas, y_train; initial_params = initial_params, opt_method = NelderMead(gas, 100))
+f = fit!(gas, y_train; initial_params = initial_params, opt_method = NelderMead(gas, 100))
```

Comment:
Why? Is `f` used for anything?

Reply:
it store the fitting statistics

Reply:
Hmm, I'm not sure I like this... I understand having a separate object for the statistics, but it's counterintuitive since `fit!` looks like it shouldn't return anything but just update `gas`. What about having the statistics inside `gas`?

Reply:
I'll approve the PR but this part kind of annoys me 😛

Reply:
We should totally open an issue to correct it